src/helpers/shared/breadcrumbs-creators.js

```diff
@@ -37,7 +38,8 @@ export const FRAGMENT_TITLE = {
   [ORDER_ROUTE]: (getState) => {
     const { portfolioItem, order } = getState().orderReducer.orderDetail;
     return `${portfolioItem.name} # ${order.id}`;
-  }
+  },
+  [PORTFOLIO_ITEM_ROUTE_EDIT]: () => 'Edit product'
```

Comment:
This should be translatable

Reply:
None of the static breadcrumbs titles are translatable yet. This will be done on its own

Reply:
Okay.